packages/feedback-service/src/typedef.graphql

```diff
@@ -0,0 +1,52 @@
+type ProfileType {
+  kerberosID: String
+  name: String
+  email: String
+}
+
+type TimestampType {
+  createdAt: DateTime
+  createdBy: ProfileType
+  modifiedAt: DateTime
+  modifiedBy: ProfileType
+}
+
+type FeedbackType {
+  _id: String
+  description: String
+  iid: String
+  experience: String
+  spa: String
+  portalFeedback: Boolean
+  timestamp: TimestampType
+  title: String
+  feedbackType: String
+}
+
+input FeedbackInput {
+  _id: String
+  description: String
+  iid: String
+  experience: String
+  module: String
+  portalFeedback: Boolean
+  timestamp: TimestampInput
+  title: String
+  feedbackType: String
+}
+
+type Query {
+  # Query Definitions
```

Comment:
This comment should describe the query/mutation on the line below it.
So here it should describe what listFeedback does.

This is helpful when using GraphQL query builders, as this is auto-documented.

Add the same for all the queries and mutations.